File: app/src/main/java/com/examples/memorizingpager_demo/MessageFragment.java
package com.examples.memorizingpager_demo;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

public class MessageFragment extends Fragment {

  public static MessageFragment newInstance(int position) {
    MessageFragment fragment = new MessageFragment();
    Bundle args = new Bundle();
    args.putInt("position", position);
    fragment.setArguments(args);
    return fragment;
  }

  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup container,
      Bundle savedInstanceState) {
    View rootView = inflater.inflate(R.layout.fragment_message, container, false);
    TextView textView = rootView.findViewById(R.id.tv_message);

    int position = getArguments().getInt("position", 0);
    textView.setText(String.valueOf(position + 1));
    return rootView;
  }

}

File: app/src/main/java/com/examples/memorizingpager_demo/MessageFragment.kt
package com.examples.memorizingpager_demo

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

class MessageFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
            val rootView = inflater.inflate(R.layout.fragment_message, container, false)
            val textView = rootView.findViewById<TextView>(R.id.tv_message)

            val position = arguments?.getInt("position", 0)
            textView.text = (position?.plus(1)).toString()
            return rootView
        }

    companion object {

        fun newInstance(position: Int): MessageFragment {
            val fragment = MessageFragment()
            val args = Bundle()
            args.putInt("position", position)
            fragment.arguments = args
            return fragment
        }
    }

}
